lib/template/config/config.rb

```diff
@@ -11,6 +11,7 @@ module Config
 
   # Mandatory -- exception is raised for these variables when missing.
   mandatory :database_url, string
+  mandatory :pliny_env, string
```

Comment:
`override` probably makes more sense.  It then behaves the same as `RAILS_ENV` etc.